.github/workflows/test.yml

```diff
@@ -25,8 +25,22 @@ jobs:
       - name: Install
         run: npm ci
 
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+
       - name: Test and generate code coverage info
         run: npm test
 
+      - name: Test Playwright and generate test report
+        run: npm run test:playwright-ct:all
```

Comment:
This means Playwright tests run in a different environment than the one on the local machine. Did you consider running it in the same Docker container also on GitHub? I'm afraid of varying test results otherwise.

See https://github.com/lmc-eu/spirit-design-system/blob/main/.github/workflows/e2e.yml#L69

Reply:
I am going to try it. I never configured Docker on Github CI, so that`s why i did not do that. And Playwright did not do that in their setup for Github CI as well. Thanks for the link

Reply:
It is now run in Docker container completely as Playwright suggests in the doc. I think that Spirit's way is not correct as it mixes setup and testing in one step. This is not best practice ...